.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+Before making this PR, please ensure the following:
+
+ - The PR is open to `dev`, NOT `master`.
+ - `CHANGELOG.md` is updated (if necessary).
+ - Documentation and tests are updated (if necessary).
```

Comment:
```suggestion
<!--
Before making this PR, please ensure the following:

 - The PR is open to `dev`, NOT `master`.
 - `CHANGELOG.md` is updated (if necessary).
 - Documentation and tests are updated (if necessary).
-->
```

This way PR authors will see the reminder but it won't be included in the PR description